### ISSUE TITLE
Update icons on side bar and cards

### ISSUE THREAD

**Issue by kshefchek** [monarch-initiative/monarch-ui#102]
Right now we use the anatomy icon as a place holder, but we need icons for publications, functions, pathways, homologs, variants, genotypes, etc.  In some cases perhaps we could reuse icons with different colors? For things such as genes, variants, and genotypes.  Individual icons are likely best if we have them.

**Comment by monicacecilia** [comment]
One place for ideas could be - https://pixabay.com/ 
For example, see here for icons for pubs https://pixabay.com/vectors/search/publications/

Of course, we also work with @jmcmurry, the Queen of design, so I'm adding her here. She may already have some icons ready for this.

**Comment by jmcmurry** [comment]
Here's a preview of draft icons fresh out of the oven. The distinction between gene, genotype, and variant is hard to iconize. Ideas welcome.

I tried to give them a consistent look and feel. Sidenote, we may want to differentiate the main parts of the menu from the (main node) facets, neighbors and overview? 

![image](https://user-images.githubusercontent.com/3658863/60688553-37dd5780-9eb6-11e9-9548-e47c766ebefd.png)

**Comment by monicacecilia** [comment]
Oooh, these are wonderful, Julie, thank you!
Maybe we could even add the files for these new icons during the mini-UI hackathon! Could I kindly ask if you could please share them with us (shared Monarch drive?), or add them yourself? Yeepee! 
The new files should replace the ones currently available here https://github.com/monarch-initiative/monarch-ui/tree/master/src/assets/img

About differentiating parts of the menu, I agree with you, we should highlight the differences. 
We could try an indentation for the list of available resources under the 'Overview' menu. And actually, I suggest that we could replace the word 'Overview' with the word 'Resources.'

**Comment by monicacecilia** [comment]
@iimpulse is interested in working on this issue - at the moment, we are waiting on files to be available.

**Comment by monicacecilia** [comment]
A few icons still pending from @jmcmurry. Will re-open to make sure we don't lose track of that. 
![julie-screenshot-icons-missing](https://user-images.githubusercontent.com/1714621/62233840-41a38f00-b37e-11e9-9809-46f5cd33338d.png)

**Comment by monicacecilia** [comment]
Sorry, re-opening until Mike has integrated them.

**Comment by jmcmurry** [comment]
Actually, @iimpulse why is pathway showing up with an anatomy icon? I didn't make a pathway icon at all but can do now.

**Comment by iimpulse** [comment]
@iimpulse i realized we also don't have a "function" one

**Comment by jmcmurry** [comment]
oh that is my fav :) Incoming

**Comment by jmcmurry** [comment]
fixed by https://github.com/monarch-initiative/monarch-ui/commit/be42412997c40f40e9471bd0260a83d3b8c10be2

**Comment by monicacecilia** [comment]
Keep this issue open until @iimpulse implements and closes with new PR.
